packages/styling/src/styles/theme.ts

```diff
@@ -96,6 +96,7 @@ function _makeSemanticColorsFromPalette(p: IPalette, isInverted: boolean): ISema
     errorText: !isInverted ? p.redDark : '#ff5f5f',
     blockingBackground: !isInverted ? 'rgba(234, 67, 0, .2)' : 'rgba(234, 67, 0, .5)',
     warningBackground: !isInverted ? 'rgba(255, 185, 0, .2)' : 'rgba(255, 251, 0, .6)',
+    warningForeground: !isInverted ? p.yellow : p.yellowLight,
```

Comment:
Let's hardcode the colors here (since yellow and yellowLight might have to swap in inverted/noninverted themes in the future).